cpp/tests/spatial/hausdorff_test.cu

```diff
@@ -32,12 +32,12 @@ using namespace cuspatial;
 
 struct is_true
```

Comment:
Rename. This is an equality comparison operator.

Reply:
It might be better to use inline lambda function as the functor is so simple. There are other similar situations. Need to figure out this. Any idea?